sdk-parsers/RMParserFramework/parsers/accuweather-parser.py

```diff
@@ -68,67 +68,72 @@ def perform(self):
                 self.parserHasData = True
-                
+
             if 'Value' in varApiDataEntry["Temperature"]["Minimum"]:
-                varEntryTemp = convertFahrenheitToCelsius(varApiDataEntry["Temperature"]["Minimum"]["Value"])
+                varEntryTemp = varApiDataEntry["Temperature"]["Minimum"]["Value"]
                 self.addValue(RMParser.dataType.MINTEMP, varEntryTime, varEntryTemp, False)
                 log.debug("TEMPERATURE MIN = %s" % (varEntryTemp))
-                self.parserHasData = True                
+                self.parserHasData = True
 
             if 'Value' in varApiDataEntry["Day"]["Wind"]["Speed"]:
-                varEntryWindSpeed = varApiDataEntry["Day"]["Wind"]["Speed"]["Value"] * 0.44704  # to meters/sec
+                varEntryWindSpeed = varApiDataEntry["Day"]["Wind"]["Speed"]["Value"] * 0.27777778 # to meters/sec
                 self.addValue(RMParser.dataType.WIND, varEntryTime, varEntryWindSpeed, False)
                 log.debug("WIND = %s" % (varEntryWindSpeed))
                 self.parserHasData = True
 
-            if 'Value' in varApiDataEntry["Day"]["SolarIrradiance"]:
-                varEntrySolarIrradiance = self.convertRadiationFromWattsToMegaJoules(varApiDataEntry["Day"]["SolarIrradiance"]["Value"])
+            if 'Value' in varApiDataEntry["Day"]["SolarIrradiance"] and 'Value' in varApiDataEntry["Night"]["SolarIrradiance"]:
+                varEntrySolarIrradiance = self.convertRadiationFromWattsToMegaJoules(varApiDataEntry["Day"]["SolarIrradiance"]["Value"] + varApiDataEntry["Night"]["SolarIrradiance"]["Value"])
                 self.addValue(RMParser.dataType.SOLARRADIATION, varEntryTime, varEntrySolarIrradiance, False)
                 log.debug("SOLARRADIATION = %s" % (varEntrySolarIrradiance))
                 self.parserHasData = True
 
             if 'Value' in varApiDataEntry["Day"]["Rain"] and 'Value' in varApiDataEntry["Night"]["Rain"]:
-                varEntryRain = convertInchesToMM(varApiDataEntry["Day"]["Rain"]["Value"] + varApiDataEntry["Night"]["Rain"]["Value"])
-                self.addValue(RMParser.dataType.RAIN, varEntryTime, varEntryRain, False)
+                varEntryRain = varApiDataEntry["Day"]["Rain"]["Value"] + varApiDataEntry["Night"]["Rain"]["Value"]
+                self.addValue(RMParser.dataType.QPF, varEntryTime, varEntryRain, False)
                 log.debug("RAIN = %s" % (varEntryRain))
                 self.parserHasData = True
-                
+
             if 'Value' in varApiDataEntry["Day"]["Evapotranspiration"] and 'Value' in varApiDataEntry["Night"]["Evapotranspiration"]:
-                varEntryEvapotranspiration = convertInchesToMM(varApiDataEntry["Day"]["Evapotranspiration"]["Value"] + varApiDataEntry["Night"]["Evapotranspiration"]["Value"])
+                varEntryEvapotranspiration = varApiDataEntry["Day"]["Evapotranspiration"]["Value"] + varApiDataEntry["Night"]["Evapotranspiration"]["Value"]

```

Comment:
It will be interesting to know if without saving ET0 the Mixer calculated ET0 is within acceptable ranges. I'm not sure how adding day/night ET0 would work in our algorithm. All our reference/summer ET0 is calculated based on Penman-Monteith equation which have day temperature min and day temperature max as variable and the delta between those reduce or increase ET0. Could be that what accuweather reports for ET just doesn't work with our precomputed summer ET0 values.